pkg/ddb/repository.go

```diff
@@ -185,6 +188,25 @@ func (r *repository) BatchGetItems(ctx context.Context, qb BatchGetItemsBuilder,
 
 		out := outI.(*dynamodb.BatchGetItemOutput)
 		responses := out.Responses[r.metadata.TableName]
+
```

Comment:
The `BatchGetItems` is growing a little bit too large.  
Can we extract some stuff into separate functions? Eg the new ttl filter stuff?